### PR TITLE
reduce log reading #3207

### PR DESCRIPTION
**Pull request by ymoctavia**
for #3207

### CODE REVIEW

**Comment by ymoctavia** [comment]
professor @damithc , previously we do not set a minimum log level for fetching logs.

Now I set the minimum level to be info level. This should help reduce the log reading.

**Comment by ymoctavia** [comment]
image

This is the log level options. Change from any level to info should reduce some unnecessary reading.

**Comment by damithc** [comment]
Yes, that should help. Also check if the cron job that periodically check the logs and email critical errors to admin. That too should be using a higher log level.

**Comment by ymoctavia** [comment]
Professor @damithc , the log compilation is also updated

File: src/main/java/teammates/logic/automated/CompileLogs.java [hunk 1]
```diff
@@ -31,7 +31,9 @@ public String doLogExam() {
         long startTime = endTime - queryRange;
 
         LogQuery q = LogQuery.Builder.withDefaults().includeAppLogs(true)
-                .startTimeMillis(startTime).endTimeMillis(endTime);
+                     .startTimeMillis(startTime).endTimeMillis(endTime)
```
Comment (damithc) [inline src/main/java/teammates/logic/automated/CompileLogs.java]: Break into two lines so that all three modifiers appear on separate lines?

**Comment by damithc** [comment]
Can go ahead and merge. 
This doesn't apply to the email log page?

**Pull request merged**